lib/chapter/event-create.js

```diff
@@ -121,187 +43,22 @@ module.exports = {
-            }
-          },
-          {
-            name: 'startDate',
-            message: 'What day will it start? (Format: YYYY/MM/DD or DD.MM.YYYY)',
-            type: 'input',
-            default: function (result) {
-              return result.startDate || input['event-start-date'] || input['event-date'] || suggestions.nextSameWeekday
-            },
-            validate: assertDate
-          },
-          {
-            name: 'startTime',
-            message: 'What time will it start? (Format: HH:MM or HH(:MM)a|pm)',
-            type: 'input',
-            default: function (result) {
-              return result.startTime || input['event-start-time'] || input['event-time'] || suggestions.lastStartTime
-            },
-            validate: assertTime
-          },
-          {
-            name: 'endDate',
-            message: 'What day will it end? (Format: YYYY/MM/DD or DD.MM.YYYY)',
-            type: 'input',
-            default: function (result) {
-              return result.startDate
-            },
-            validate: function (raw, data) {
-              var result = assertDate(raw)
-              if (result === true) {
-                var duration = eventTime.forEvent({
-                  startDate: data.startDate,
-                  startTime: '00:00',
-                  endDate: raw,
-                  endTime: '00:00',
-                  location: {
-                    timeZone: 'UTC'
-                  }
-                }).durationInMs
-                if (duration < 0) {
-                  return 'The end date needs to be at the same day or after the start date.'
-                } else {
-                  return true
-                }
-              }
-              return result
-            }
-          },
-          {
-            name: 'endTime',
-            message: 'What time will it end? (Format: HH:MM or HH(:MM)a|pm)',
-            type: 'input',
-            default: function (result) {
-              return result.endTime
-            },
-            validate: function (raw, data) {
-              var result = assertTime(raw)
-              if (result === true) {
-                var eT = eventTime.forEvent({
-                  startDate: data.startDate,
-                  startTime: data.startTime,
-                  endDate: data.endDate,
-                  endTime: raw,
-                  location: {
-                    timeZone: 'UTC' // we need a timezone, but it doesn't matter which
-                  }
-                })
-                var duration = eT.durationInMs
-                if (duration <= 0) {
-                  return 'The end time needs to be after the start time.'
-                } else {
-                  return true
-                }
-              }
-              return result
-            }
-          }
+          eventHelper.askForUrl,
+          eventHelper.askForName(input, suggestions, locations),
```

Comment:
Couldn't it be even better if `askForName`(etc.) would not take `input` but rather the `input` properties that they need?